deny.toml

```diff
@@ -22,6 +22,7 @@ allow = [
     "MIT-0",
     "MPL-2.0",
     "Unicode-DFS-2016",
+    "WTFPL",
```

Comment:
Crate pcx is licensed WTFPL: https://en.wikipedia.org/wiki/WTFPL

Certainly for me, it's an acceptable license, so I've changed it here, but this would need project approval, of course.

If there's resistance to adding this dependency, I could look at a from-scratch PCX implementation, which would also give an opportunity to fix the decoder bugs and memory inefficiency around paletted images, perhaps.

Reply:
I'm fine with it, I guess? I've chosen it before for projects. That said, I think we should very explicitly document these in both locations, due to corporate users that do not intent to introduce such dependencies: a) at the feature location that enabling it introduces an unusual `WTFPL` license b) at the allow list that it comes from a feature. If we could have deny check (b) automatically in some way, even better.

Maybe allow _just_ that crate? [As in this example here](https://embarkstudios.github.io/cargo-deny/checks/licenses/cfg.html#example) — an old `adler32` crate.

Reply:
Thanks. Is https://github.com/image-rs/image/pull/2364/commits/25cede1450143eb797066d56eca3e66e4d16c249 sufficient, do you think, or should it be called out in the README, or in more detail?

Reply:
LGTM.